monitoring/monitoring.go

```diff
@@ -220,12 +221,21 @@ func WithPrometheusImage(image string) Option {
 	}
 }
 
+func WithUseCadvisor(use bool) Option {
```

Comment:
I would perhaps turn this around, and make this option `WithCadvisorDisabled`, I think we'll want to have cAdvisor enabled by default, but give users the option to disable it. What do you think?

Reply:
Thanks for the feedback @matej-g . Yh I considered this and just ended up deciding to explicitly set it to true inside the `Start` function. I think this still maintains backwards compatibility. I could still change it if you strongly prefer the flip.

Reply:
I agree with Matej. To me reasons are:

"With Use Cadvisor" does not sounds like a valid english sentence. We have opportunity to make this readable by making "WithCadvisorDisabled"

We could do `WithCadvisor(use bool)`, but bool have two option and we already by default have TRUE. So just "disabled".

Reply:
Thanks. Pushed a fix!